readthedocs/builds/constants.py

```diff
@@ -61,3 +61,37 @@
     LATEST,
     STABLE,
 )
+
+# GitHub Build Statuses
+GITHUB_BUILD_STATE_FAILURE = 'failure'
+GITHUB_BUILD_STATE_PENDING = 'pending'
+GITHUB_BUILD_STATE_SUCCESS = 'success'
+
+# GitHub Build Statuses
```

Comment:
```suggestion
# GitLab Build Statuses
```

Reply:
Also, these aren't used currently are they?

Reply:
No, They are not but we will use it in future. should I remove them for now?

Reply:
Yes.